### ISSUE TITLE
May the doc be wrong?

### ISSUE THREAD

**Issue by luisesn** [williamtroup/Calendar.js#146]
After using the example code:
```
  var calendarElement = document.getElementById( "calendar" );
  var calendarInstance2 = calendarJs( calendarElement, {
    manualEditingEnabled: true
    // All your options can be set here
  } ); 
```
calendarInstance2 is undefined

It works by using "new"
```
  var calendarElement = document.getElementById( "calendar" );
  var calendarInstance2 = new calendarJs( calendarElement, {
    manualEditingEnabled: true
    // All your options can be set here
  } ); 
```

**Comment by williamtroup** [comment]
Hi,

Thank you for letting me know.  Will get this updated for the next release.

**Comment by williamtroup** [comment]
I've just released v2.3.3, which contains the documentation fixes.

Thanks for the report!

Don't forget to Star and Share the project!
